.vscode/settings.json

```diff
@@ -1,5 +1,5 @@
 {
   "eslint.enable": true,
-  "eslint.autoFixOnSave": true,
+  "eslint.autoFixOnSave": false,
```

Comment:
Sorry, I didn't spot this before. Is this correct? I thought we were intentionally running Prettier on save to ensure JS code conforms with our conventions.